### PR TITLE
Update caching logic

### PR DESCRIPTION
**Pull request by jj105**
# fix issues in caching logic

### CODE REVIEW

File: girderformindlogger/utility/jsonld_expander.py [hunk 1]
```diff
@@ -816,19 +816,24 @@ def formatLdObject(
                 'http://schema.org/url',
                 obj.get('meta', {}).get('protocol', obj).get('url')
             )
+
+            # get protocol data from url (load from database unless refreshCache is set to True)
```
Comment (devbtech) [inline girderformindlogger/utility/jsonld_expander.py]: I think it's better don't use ternary operator here
initialize your variable and it to empty dict

like that for example:
```
protocol = {}
if protocolUrl is not None:
      protocol = ProtocolModel().getFromUrl(
                protocolUrl,
                'protocol',
                user,
                thread=False,
                refreshCache=refreshCache
            )[0]
```

File: girderformindlogger/models/model_base.py [hunk 2]
```diff
@@ -261,10 +261,10 @@ def getFromUrl(
             raise ResourcePathNotFound("Document not found: {}".format(str(
                 passedUrl
             )))
-        atType, cachedDoc = cycleModels(
+        cachedDoc = cycleModels(
```
Comment (devbtech) [inline girderformindlogger/models/model_base.py]: The same here
just try don't use ternary operator if it possible

**Comment by shnizzedy** [comment]
@devbtech why is ([PEP 308](https://www.python.org/dev/peps/pep-0308/)) ternary worse than non-ternary?

**Comment by henryrossiter** [comment]
> @devbtech why is ([PEP 308](https://www.python.org/dev/peps/pep-0308/)) ternary worse than non-ternary?

Personally, I find some of the longer, multi-line ternary statements to be difficult to read.

I am also curious to hear @devbtech 's opinion

**Comment by devbtech** [comment]
Ternary operations look like a mess in the code. Especially when to use multiple nesting. (I’m talking specifically for this and some cases in code not about the operator as a whole)
there are hundreds of places worth rewriting in the code.

this not only impairs readability, it also reduces the chance of debugging

**Comment by jj105** [comment]
The main reason I used it is that I wanted to maintain coding style over all code.
I'll update and resubmit PR if we need it.

**Pull request merged**